modules/web/src/app/shared/components/side-nav-field/style.scss

```
@@ -0,0 +1,57 @@
+// Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@use 'variables';
+
+.side-nav-bar {
+  margin-left: 10px;
+}
+
+.expanded {
+  border-radius: variables.$border-radius;
+}
+
+.expanded-menu {
+  border-radius: variables.$border-radius;
+}
+
+button {
+  cursor: pointer;
+  margin: 5px 15px 10px 10px;
+  padding: 0 0 0 20px;
+
+  .mat-menu-trigger {
+    left: 20px;
+    position: relative;
+    top: -10px;
+  }
+}
+
+.nav-field-label {
+  cursor: pointer;
+  height: 45px;
+  margin: 0 0 15px;
+  padding: 10px;
+
+  i:last-child {
+    font-size: variables.$font-size-subhead;
+    position: absolute;
+    right: 10px;
+    text-decoration: none;
```

Positioning can be handled with flex layout (see comment on HTML file)

```suggestion
  i:last-child {
    font-size: variables.$font-size-subhead;
    text-decoration: none;
```